capplets/display/scrollarea.h

```diff
@@ -18,6 +18,9 @@
 #include <cairo.h>
 #include <gtk/gtk.h>
 
+#ifndef __SCROLLAREA_H__
```

Comment:
Move to the first line at the beginning of the code

Reply:
done